mod-audit-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -79,6 +79,11 @@
       "run": "after",
       "snippetPath": "acquisition/create_acquisition_piece_log_table.sql",
       "fromModuleVersion": "mod-audit-2.9.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "acquisition/create_acquisition_invoice_log_table.sql",
+      "fromModuleVersion": "mod-audit-2.10.1"
```

Comment:
should be 3.0.0
Also please modify to 3.0.0-SNAPSHOT here https://github.com/folio-org/mod-audit/blob/master/pom.xml#L6